.travis.yml

```diff
@@ -20,24 +20,21 @@ env:
 
 before_install:
   - pip install --no-cache-dir -U -r requirements-test.txt
-  - openwisp-utils-qa-checks --migration-path "./django_ipam/migrations/"
-  - openwisp-utils-qa-checks --migration-path "./tests/sample_ipam/migrations/" --skip-checkcommit --skip-checkendline --skip-flake8 --skip-isort
   - npm install -g jslint
   - jslint ./django_ipam/static/django-ipam/js/*.js
 
 install:
   - pip install "django>=2.0,<2.3"
   - python setup.py -q develop
 
-before_script:
+script:
+  - >
+    openwisp-utils-qa-checks --migration-path "./django_ipam/migrations/ ./tests/sample_ipam/migrations/"
+    --skip-checkmakemigrations
   - |
-    if [[ $TRAVIS_PYTHON_VERSION == 3.5 && $SAMPLE_APP != "1" ]]; then
-      OUTPUT=$(./tests/manage.py makemigrations --dry-run django_ipam)
-      echo $OUTPUT
-      echo $OUTPUT | grep "No changes detected"
+    if [[ $TRAVIS_PYTHON_VERSION >= 3.5 && $SAMPLE_APP != "1" ]]; then
```

Comment:
I just saw travis failing:
I think:
```
TRAVIS_PYTHON_VERSION >= 
```
is not allowed, can you please try `==` to fix it ? 
Use python version 3.6 ! :smile: